db/migrate/20220719180225_add_sort_index_to_school_links.rb

```diff
@@ -0,0 +1,5 @@
+class AddSortIndexToSchoolLinks < ActiveRecord::Migration[6.1]
+  def change
+    add_column :school_links, :sort_index, :integer, default: 0, null: false
+  end
+end
```

Comment:
This will need a data migration for populating the sort index for the existing school links.